server/src/test/java/org/opensearch/index/store/RemoteSegmentStoreDirectoryTests.java

```diff
@@ -1141,6 +1141,62 @@ public void testMetadataFileNameOrder() {
         assertEquals(14, count);
     }
 
+    public void testInitializeToSpecificTimestampNoMetadataFiles() throws IOException {
```

Comment:
We moved the tests here as these should work with pinned timestamp setting enabled as well as disabled